app/layout.tsx

```diff
@@ -19,10 +18,15 @@ export default function RootLayout({
   return (
     <html lang="en">
       <head>
-        <ColorSchemeScript />
```

Comment:
[Matine docs](https://mantine.dev/theming/color-schemes/#colorschemescript) note:

> `ColorSchemeScript` . . . is used to avoid flash of inaccurate color scheme in server side rendered applications, for example Next.js or Remix.

Is `ColorSchemeScript` compatible with the theming you implemented?

Reply:
It was causing a bunch of styling overwriting and oddities. A lot of the documentation I read had talked about switching to HeadlessMantineProvider, but that created new issues as I was working. When I read through [the documentation here](https://v7.mantine.dev/theming/color-schemes) it didn't seem to be necessary to include it. If there's a better way of handling this, please let me know.